Scrapy-basics/scrapy_basics/scrapy_basics/middlewares.py

```diff
@@ -0,0 +1,103 @@
+# Define here the models for your spider middleware
```

Comment:
Seems like this module is not used in your project

Reply:
I removed it from the repo.